pom.xml

```diff
@@ -63,6 +63,11 @@
             <artifactId>guava</artifactId>
             <version>16.0.1</version>
         </dependency>
+        <dependency>
+            <groupId>org.roaringbitmap</groupId>
+            <artifactId>RoaringBitmap</artifactId>
+            <version>0.5.18</version>
```

Comment:
explained here: https://github.com/metamx/extendedset/pull/10#discussion_r90695147. As this repository becomes obsolete, I didn't intend to make the "most maintainable and right" decisions, rather quick and beneficial for execution speed. When merged into Druid, it should probably be solved differently.